### DISCUSSION TITLE
Некорректный перевод

### DISCUSSION THREAD

**Discussion by flameshikari** [flameshikari/outline-ru#8] [Перевод]
Здесь вы можете сообщить о некорректно переведённых строках и как их можно улучшить.

**Comment by antitoxicic** [comment]
"Home": "Главная",
я бы перевел как "Домашняя" или лучше "Недавнее" -> так как лучше отражает суть, в это вкладке показываются только недавние изменения

**Reply by flameshikari** [reply]
Раньше было «Домашняя страница», но я решил укоротить строку до «Главная». Даже как-то не подумал про «Домашняя». Сделаю тогда «Домашняя» к следующему релизу; так более дословный перевод получится.

**Reply by sovushik** [reply]
Как раз свежая версия с ура вышла - 0.85.0
